From c71287fa6aacdf24a3d8a24dae5c4c6a675d24da Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Barnab=C3=A1s=20Cz=C3=A9m=C3=A1n?=
 <barnabas.czeman@mainlining.org>
Date: Thu, 2 Apr 2026 00:29:37 +0200
Subject: [PATCH 19/22] arm64: dts: qcom: sm6125-xiaomi-laurel-sprout: Remove
 extcon
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

GPIO pin 102 is related to DisplayPort what is not supported
by this device and it is also disabled at downstream,
remove the unnecessary extcon-usb node

Signed-off-by: Barnabás Czémán <barnabas.czeman@mainlining.org>
---
 arch/arm64/boot/dts/qcom/sm6125-xiaomi-laurel-sprout.dts | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/sm6125-xiaomi-laurel-sprout.dts b/arch/arm64/boot/dts/qcom/sm6125-xiaomi-laurel-sprout.dts
index 944c7ae38..910a7c086 100644
--- a/arch/arm64/boot/dts/qcom/sm6125-xiaomi-laurel-sprout.dts
+++ b/arch/arm64/boot/dts/qcom/sm6125-xiaomi-laurel-sprout.dts
@@ -62,11 +62,6 @@ cmdline_mem: memory@ffd00000 {
 		};
 	};
 
-	extcon_usb: usb-id {
-		compatible = "linux,extcon-usb-gpio";
-		id-gpios = <&tlmm 102 GPIO_ACTIVE_HIGH>;
-	};
-
 	gpio-keys {
 		compatible = "gpio-keys";
 
@@ -603,10 +598,6 @@ &usb3 {
 	status = "okay";
 };
 
-&usb3_dwc3 {
-	extcon = <&extcon_usb>;
-};
-
 &usb_dwc3_hs {
 	remote-endpoint = <&pmi632_hs_in>;
 };
-- 
2.56.0

